signer/src/bitcoin/rpc.rs

```diff
@@ -731,3 +734,107 @@ impl BitcoinInteract for BitcoinCoreClient {
         self.get_network_info()
     }
 }
+
+/// Assesses the total fees paid for any outstanding sweep transactions in
+/// the mempool which may need to be RBF'd. If there are no sweep
+/// transactions which are spending the signer's UTXO, then this function
+/// will return [`None`].
+///
+/// TODO: This method currently blindly assumes that the mempool transactions
```

Comment:
Could you create an issue and add it to the TODO? I think it sounds reasonable to have a sanity check..